platforms/anchor-platform/admin-guide/sep6/configuration.mdx

```diff
@@ -176,7 +180,16 @@ SECRET_SEP6_MORE_INFO_URL_JWT_SECRET="your encryption key shared with your busin
 
 </CodeExample>
 
-See the [KYC API][platform-api-kyc] and [Rates API][sep38] for details on the endpoints that must be implemented on your business server.
+Businesses can set a deadline for user actions on transactions using the `user_action_required_by` field. For examples, see [JSON-RPC Methods][json-rpc-methods]. Alternatively, they can use the `initial_user_deadline_seconds` parameter to define a default time period (in seconds)  to help populate the deadline timestamp.
```

Comment:
This methods are not alternative (mutually exclusive), they should be used in combination. Can you reword that?
`Alternatively` -> `In addition,`
And let's not use "they" (thinking of a person or people), we can use business/anchor or passive voice (thinking of a server).